spa/proxy.conf.json

```diff
@@ -1,6 +1,13 @@
 {
+
+  "/api/v1": {
+    "target": "http://localhost:3001",
+    "secure": false,
+    "debug":true
+  },
   "/api": {
     "target": "http://localhost:3000",
     "secure": false
+
```

Comment:
Just checking, because I don't have the full picture for this in my head -- should `/api/v1` and `/api` go to different ports?

Reply:
👍